### ISSUE TITLE
grep options under OpenBSD

### ISSUE THREAD

**Issue by TheLinuxKitten** [robbyrussell/oh-my-zsh#3470]
Under OpenBSD '--color' is not a valid option for grep. The alias for grep in https://github.com/robbyrussell/oh-my-zsh/blob/master/plugins/common-aliases/common-aliases.plugin.zsh#L18 causes grep commands to fail always.

**Comment by hobarrera** [comment]
+1

Just hitting enter on an empty zsh line results in two error (due to grep being used by themes):

```
$ 
grep: unknown option -- color=auto
usage: grep [-abcEFGHhIiLlnoqRsUVvwxZ] [-A num] [-B num] [-C[num]]
        [-e pattern] [-f file] [--binary-files=value] [--context[=num]]
        [--line-buffered] [pattern] [file ...]
grep: unknown option -- color=auto
usage: grep [-abcEFGHhIiLlnoqRsUVvwxZ] [-A num] [-B num] [-C[num]]
        [-e pattern] [-f file] [--binary-files=value] [--context[=num]]
        [--line-buffered] [pattern] [file ...]
```

**Comment by mcornella** [comment]
Ok we'll change that. What about in `lib/grep.zsh`, is it working ok that definition?

**Comment by hobarrera** [comment]
> Ok we'll change that. What about in lib/grep.zsh, is it working ok that definition?

I don't think I understood the question; can you rephrase that?

**Comment by mcornella** [comment]
Yeah sorry :grin: The grep alias is also defined in file `lib/grep.zsh`, can you check that it works ok in your system?

**Comment by hobarrera** [comment]
The mentioned script works - but defines an alias with a flag that does not exist on OpenBSD's grep: --color.

**Comment by TheLinuxKitten** [comment]
Ohps!! @hobarrera, I've not seen your comment and pull request.

**Comment by hobarrera** [comment]
Hah, I'll consider this a favourable code review. ^_^

**Comment by pawciobiel** [comment]
You could also do a dependency on sysutils/ggrep
